doc/Caliptra.md

```diff
@@ -511,58 +511,47 @@ See [HW Section](#hardware-section) for a detailed discussion.
 
 ## Caliptra Profiles
 
-Caliptra supports two physical modes of instantiation, the Caliptra Core and the Caliptra Subsystem.  Within Caliptra Core there are two modes of integration with different security postures and firmware loading techniques.  The first is Boot Media Integrated Profile (formerly Active Profile) where Caliptra loads its firmware directly from persistent storage (eg. SPI) while in Boot Media Dependent Profile (formerly Passive Profile) Caliptra firmware is pushed into mailbox SRAM buffer by the SoC immutable code (HW or ROM) that controls the persistent storage of the firmware code.
+Caliptra supports two physical modes of instantiation, the Caliptra Core and the Caliptra Subsystem.  Within Caliptra Core there are two modes of integration with different security postures and firmware loading techniques.
 
-### <a id="bmi"></a>Caliptra Core - Boot Media Integrated (BMI)
+1. Boot Media Integrated (BMI) Profile (formerly Active Profile): Caliptra loads its firmware directly from SPI boot media. SoC boot cannot proceed through any other boot media.
```

Comment:
Is the goal here that all SOC FW comes from through Caliptra or just to ensure that Caliptra gets its own FW?

Reply:
That is not closed and is one of the reasons 1.0 won't be supporting the BMI Profile.

Reply:
Understood